src/plugile.rpgle

```diff
@@ -5315,5 +5331,23 @@
        return rcb;
       /end-free
      P                 E
+      *****************************************************
+      * set runing mode in 32-bit or 64-bit
```

Comment:
I'm not sure getters and setters are really necessary here.

Reply:
plugpase.rpgle needs the declarations of those 2 procedures to set the pointer size globally.

Reply:
But why is a function needed instead of just referring to the variable itself?

Reply:
plugpase.rpgle includes plugconf_h.rpgle to use the declarations of getter/setter. If put the variable directly in the plugconf_h.rpgle, then plugpgase.rpgle includes the header. We got 2 separate copies of that variable.